src/bp/core/services/dialog/dialog-engine.ts

```diff
@@ -70,7 +70,7 @@ export class DialogEngine {
         return this.processEvent(sessionId, event)
       } else if (result.followUpAction === 'wait') {
         // We don't call processEvent, because we want to wait for the next event
-        this._logDebug(event.botId, event.target, 'waiting until next event')
+        this._logDebug(event.botId, event.target, 'waiting until next event', { ...event.state.context })
```

Comment:
since the dialog debug is enabled by default, I think that level of details should be optional, context can get quite big and logs will be hard to read, for example:

```js
{
"currentFlow":"skills/choice-62d44d.flow.json",
"currentNode":"parse",
"previousFlow":"main.flow.json",
"previousNode":"topic-choices",
"jumpPoints":[{"flow":"main.flow.json","node":"topic-choices"}],
"queue":{"instructions":[
{"type":"on-receive","fn":"basic-skills/choice_parse_answer {\"contentId\":\"builtin_single-choice-mrFFU_\",\"keywords\":{\"context\":[\"context\",\"What is a Context?\"],\"memory\":[\"memory\",\"What is Bot Memory?\"]},\"config\":{}}"},
{"type":"transition","fn":"temp['skill-choice-valid'] === true","node":"#"},{"type":"transition","fn":"true","node":"invalid"}]}}
```